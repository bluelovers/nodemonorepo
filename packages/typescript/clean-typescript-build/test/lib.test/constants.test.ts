import { TARGETED_EXTENSIONS } from '../../index'

it('TARGETED_EXTENSIONS matches snapshot', () => {
  expect(new Set(TARGETED_EXTENSIONS)).toMatchSnapshot()
})
