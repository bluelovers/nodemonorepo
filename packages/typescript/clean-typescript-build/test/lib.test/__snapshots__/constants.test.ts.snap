// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TARGETED_EXTENSIONS matches snapshot 1`] = `
Set {
  ".js",
  ".js.map",
  ".jsx",
  ".jsx.map",
  ".mjs",
  ".mjs.map",
  ".d.ts",
  ".d.ts.map",
  ".d.tsx",
  ".d.tsx.map",
  ".js.d.ts",
  ".js.d.ts.map",
  ".jsx.d.ts",
  ".jsx.d.ts.map",
  ".mjs.d.ts",
  ".mjs.d.ts.map",
}
`;
